tests/integration/article.test.js

```diff
@@ -38,4 +31,42 @@ describe('POST /api/v1/articles', () => {
         done();
       });
   });
+
+  after(async (done) => {
+    app.close();
+    app = null;
+    done();
+  });
+});
+
+describe('PATCH /api/v1/articles/:id', () => {
+  beforeEach(async () => {
+    app = await startServer(5000);
+    agent = chai.request(app);
+  });
+  it('Should return status: 202', (done) => {
+    agent.patch('/api/v1/articles/979eaa2e-5b8f-4103-8192-4639afae2ba8')
+      .send(UPDATED_ARTICLE)
+      .end((_err, res) => {
+        const { body } = res;
+        expect(res).to.have.status(202);
+        expect(body).should.be.an('object');
+        expect(body).to.have.property('status');
+        expect(body).to.have.property('message');
+        expect(body).to.have.property('data');
+        expect(body.status).to.equal(true);
+        expect(body.message).to.be.a('string');
+        expect(body.data).to.be.an('array');
+        expect(body.data[0].title).to.be.equal(ARTICLE.title);
+        expect(body.data[0]).to.have.property('slug');
+        expect(body.data[0].slug).to.be.a('string');
```

Comment:
You can improve on the test. When you update an article, a valid test would be to get that updated and check what was updated to be correct.